paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsItem.kt

```diff
@@ -5,21 +5,21 @@ import com.stripe.android.model.PaymentMethod
 internal sealed class PaymentOptionsItem {
 
     abstract val viewType: ViewType
-    abstract val isEnabledDuringEditing: Boolean
+    abstract fun isEnabledDuringEditing(useUpdatePaymentMethodScreen: Boolean): Boolean
```

Comment:
When we GA this screen, is this parameter going to be removed?

Reply:
Just removed it -- I was originally thinking we weren't going to update the customer session experience as part of this, which is why I originally added this param. But I thought about it more, and for consistency, customer sheet should have the same navigation experience as payment sheet horizontal mode. Made some reasonably large changes to this PR to support that